<launch>
  <arg name="input_topic_name" default="/sensing/camera/camera0/image_raw"/>
  <arg name="input/twist" default="/sensing/vehicle_velocity_converter/twist_with_covariance"/>
  <!-- Parameter -->
  <arg name="param_file" default="$(find-pkg-share autoware_image_diagnostics)/config/image_diagnostics_node.param.yaml"/>
  <node pkg="autoware_image_diagnostics" exec="image_diagnostics_node" name="image_diagnostics_node">
    <remap from="input/raw_image" to="$(var input_topic_name)"/>
    <remap from="input/twist" to="$(var input/twist)"/>
    <param from="$(var param_file)"/>
  </node>
</launch>
